openassessment/assessment/test/test_team.py

```diff
@@ -0,0 +1,327 @@
+"""
+Tests for team assessments.
+"""
+from __future__ import absolute_import
+
+import mock
+from freezegun import freeze_time
+
+from django.utils.timezone import now
+
+from submissions import team_api as team_submissions_api
+
+from openassessment.assessment.api import teams as teams_api
+from openassessment.assessment.models.staff import TeamStaffWorkflow
+from openassessment.assessment.test.constants import OPTIONS_SELECTED_DICT, RUBRIC
+from openassessment.tests.factories import TeamStaffWorkflowFactory, UserFactory
+from openassessment.test_utils import CacheResetTest
+
+
+STAFF_TYPE = "ST"
+
+
+class TestTeamApi(CacheResetTest):
+    """ Tests for the Team Assessment API """
+
+    def setUp(self):
+        super().setUp()
+
+        # Users for testing
+        staff_user = UserFactory.create()
+        staff_user.is_staff = True
+        staff_user.save()
+        self.staff_user_id = staff_user.id
+
+        self.submitting_user_id = UserFactory.create().id
+        team_member_1_id = UserFactory.create().id
+        team_member_2_id = UserFactory.create().id
+
+        self.team_member_ids = [self.submitting_user_id, team_member_1_id, team_member_2_id]
+
+        self.default_assessment = (
+            self.staff_user_id,  # scorer_id
+            OPTIONS_SELECTED_DICT["few"]["options"],  # options_selected
+            dict(),  # critereon_feedback
+            '',  # overall_feedback
+            RUBRIC  # rubric_dict
+        )
+
+    def test_submitter_is_finished(self):
```

Comment:
For future, consider using `setUpTestData` method: https://docs.djangoproject.com/en/1.11/topics/testing/tools/#django.test.TestCase.setUpTestData . That way, any objects you create will be rolled back once this test class is finished.

Reply:
Ooh, good point. I think I'll move the user creation into there, but keep the assessments in the `setUp` function so that we have clean data for each run.